package.json

```
@@ -3,6 +3,18 @@
   "version": "0.0.0-development",
   "description": "OpenApi TypeScript client generator",
   "main": "lib/index.js",
+  "exports": {
+    "./lib/runtime": {
+      "require": "./lib/runtime/index.js",
+      "default": "./lib/runtime/index.js",
+      "types": "./lib/runtime/index.d.ts"
+    },
+    "./lib/runtime/query": {
+      "require": "./lib/runtime/query.js",
+      "default": "./lib/runtime/query.js",
+      "types": "./lib/runtime/query.d.js"
+    }
```

Perhaps this can now be simplified so that the imports are always done from the root:

```ts
import * as Oazapfts from "oazapfts/runtime";
import * as QS from "oazapfts/runtime/query";
```

```suggestion
    "./runtime": {
      "require": "./lib/runtime/index.js",
      "default": "./lib/runtime/index.js",
      "types": "./lib/runtime/index.d.ts"
    },
    "./runtime/query": {
      "require": "./lib/runtime/query.js",
      "default": "./lib/runtime/query.js",
      "types": "./lib/runtime/query.d.js"
    }
```

---

I like this. Will only prepare the exports for this though and add a note for next major. This might break in old environments that do not respect the `exports` field